sdk/identity/identity/interactive-browser-credential.md

```diff
@@ -56,7 +46,6 @@ Then you need to visit your app registration and update the redirect URI you're
 
 ## Sample code
 
-You can see a sample project that uses `InteractiveBrowserCredential` with both [Authorization Code Flow][AuthCodeFlow] and [Implicit Grant Flow][ImplicitGrantFlow] here: [link to the sample project](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/identity/identity/test/manual).
+You can see a sample project that uses `InteractiveBrowserCredential` with the [Authorization Code Flow][AuthCodeFlow] here: [link to the sample project](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/identity/identity/test/manual).
```

Comment:
Do we even need to talk about different flows now that we only support the one?

Reply:
Good point! I'll remove it where it's no longer necessary!